### DISCUSSION TITLE
Last Nightly Mac OS - can't open application

### DISCUSSION THREAD

**Discussion by Newcal** [supermerill/SuperSlicer#2733] [Q&A]
The app seems too be broken : "You can't open the application
"SuperSlicer" because it may be
damaged or incomplete."

The .dmg seems lighter than usual : around 40mb instead of 50+mb usually.
Is it expected or is there any others to fix this ?

Thank you.

**Comment by Thormus** [comment]
I don't think its working on an m1 mac.

**Comment by supermerill** [comment]
if the linux build was broken near it, it's effectively a  broken build.

or https://github.com/supermerill/SuperSlicer/issues/2543

**Reply by Newcal** [reply]
Last version tested. Now, I am able to open it but it does crash like weeks ago after loading it.

Thanks mate for your work :)
